README.md

```diff
@@ -192,29 +220,43 @@ cd tdx/guest-tools
 TD_IMG=<path_to_td_qcow2_image> ./run_td_virsh.sh
 ```
 
-## Verify TD Guest
+<a id="verify-td-guest"></a>
+## 7. Verify TD Guest
+
+1. Confirm a TD guest is running.
```

Comment:
This is not a very accurate method of verification, user may have an other instance of running QEMU that is not related to TD. I dont think we need to give users this instructions, if users cannot ssh into the guest, that means the guest is not running,

Reply:
ok.  removed.